vNext/extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -544,14 +546,15 @@ export class AjaxMonitor implements ITelemetryPlugin, IDependenciesPlugin, IInst
                         responseFinishedTime: ajaxData.responseFinishedTime
                     });
             } else {
-                let dependency: IDependencyTelemetry = {
+                let dependency= <IDependencyTelemetry>{
                     id: ajaxData.id,
                     target: ajaxData.getAbsoluteUrl(),
                     name: ajaxData.getPathName(),
                     type: "Fetch",
                     duration: ajaxData.ajaxTotalDuration,
                     success: false,
                     responseCode: 0,
+                    time: ajaxData.requestSentTime,
```

Comment:
Why do you need to override time of the event?

Reply:
We are currently generating the timestamp when the telemetry item is generated, instead of when the request starts. This is inconsistent with how other SDKs handle this. #628 has further discussion on this

Reply:
Ok, so it will be better to actually call this property sentTime or somethign like that so people calling the APIs directly don't get confused of what time means here

Reply:
renamed to `sentTime`